Bearded.Graphics/Bearded.Graphics.csproj

```diff
@@ -10,12 +10,11 @@
 
   <ItemGroup>
     <PackageReference Include="Bearded.Utilities" Version="0.2.0.393-dev" />
-    <PackageReference Include="OpenTK.Graphics" Version="4.4.0" />
-    <PackageReference Include="OpenTK.Windowing.Common" Version="4.4.0" />
-    <PackageReference Include="OpenTK.Windowing.Desktop" Version="4.4.0" />
-    <PackageReference Include="OpenTK.Windowing.GraphicsLibraryFramework" Version="4.4.0" />
```

Comment:
Needed to update either these or the reference in Bearded.Graphics.System.Drawing to make the example work. Felt wrong to downgrade the other so I upgraded this one.

After merging this, I'll submit another that pushes the version to 4.7.5 for all projects (latest stable).

Reply:
Probably dependabot will start picking up these changes now that the solution file is up to date again. You can always force a check on that as well.